src/NFTAdapter.sol

```diff
@@ -49,27 +49,24 @@ contract NFTAdapter is DSNote {
     bytes12 public kin;
     GemLike public gem;
 
-    int256 constant ONE = 10 ** 45; // rad
-
     constructor(address vat_, bytes12 kin_, address gem_) public {
         vat = VatLike(vat_);
         kin = kin_;
         gem = GemLike(gem_);
     }
 
-    function join(bytes32 urn, uint256 obj) external note {
+    function join(address urn, uint256 obj) external note {
         require(uint256(uint160(obj)) == obj, "obj-overflow");
 
         gem.transferFrom(msg.sender, address(this), obj);
-        vat.slip(ilkName(kin, obj), urn,  ONE);
+        vat.slip(ilkName(kin, obj), urn,  1);
     }
 
-    function exit(bytes32 urn, address guy, uint256 obj) external note {
+    function exit(address guy, uint256 obj) external note {
```

Comment:
I think we use `usr` now on dss, so should we use it here too?

Reply:
Oh yes, it is using `usr` in the other ones. Let's update it.

Reply:
Done!